dual_net.py

```diff
@@ -157,13 +103,93 @@ def __init__(self,
         self.update_op = tf.train.AdamOptimizer(learning_rate).minimize(self.loss)
         self.sess = sess
 
+
+    def build_model(self,
```

Comment:
Why did you move this into the class? It doesn't seem to use `self` anywhere

Reply:
It used to with another approach I was trying, you're right it doesn't need to be there anymore.